lisa/operating_system.py

```diff
@@ -198,19 +236,46 @@ def name_pattern(cls) -> Pattern[str]:
     def _initialize_package_installation(self) -> None:
         self._node.execute("apt-get update", sudo=True)
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> ExecutableResult:
         command = (
             f"DEBIAN_FRONTEND=noninteractive "
             f"apt-get -y install {' '.join(packages)}"
         )
-        self._node.execute(command, sudo=True)
+        if not signed:
+            command = command.__add__(" --allow-unauthenticated")
+
+        return self._node.execute(command, sudo=True)
+
+    def _get_os_version(self) -> OsVersion:
+        os_version = OsVersion("")
+        cmd_result = self._node.execute(cmd="which lsb_release", no_error_log=True)
+        if cmd_result.exit_code != 0 and cmd_result.stdout != "":
+            os_version.vendor = self._node.execute(
+                cmd="lsb_release -i -s", no_error_log=True
+            ).stdout
+            os_version.release = self._node.execute(
+                cmd="lsb_release -r -s", no_error_log=True
+            ).stdout
+            os_version.release = self._node.execute(
```

Comment:
why the release is set twice?

Reply:
Isn't all of that also present on /etc/os-release, for all of the distros mentioned here? We can populate all that in a single query/read on that.

Reply:
> Isn't all of that also present on /etc/os-release, for all of the distros mentioned here? We can populate all that in a single query/read on that.

yup, adding for /etc/os-release and only if it didnt work for maybe like old distros then can check by other means